feeder/src/vote.ts

```diff
@@ -303,20 +303,21 @@ interface VoteArgs {
 
 export async function vote(args: VoteArgs): Promise<void> {
   const client = new LCDClient({
-    URL: args.lcdAddress,
-    chainID: args.chainID,
-    gasPrices: args.gasPrices,
+    URL: process.env['LCD_ADDRESS'] || args.lcdAddress,
+    chainID: process.env['CHAIN_ID'] || args.chainID,
+    gasPrices: process.env['GAS_PRICE'] || args.gasPrices,
```

Comment:
I think in general arguments on command line should have higher precedence than env vars, so I think you should swap them around. Same goes for changes further down in this file.

Reply:
I agree on this. Could you apply this to your PR? @joe-bowman